Doc/library/sys.rst

```diff
@@ -449,10 +449,12 @@ always available.
 
 .. function:: exit([arg])
 
-   Exit from Python.  This is implemented by raising the :exc:`SystemExit`
+   Exit from Python.  If called in the main thread, this raises the :exc:`SystemExit`
```

Comment:
I think this always raises SystemExit, just the impact of that changes. (Check the code).

‘Exit from Python.’ Can be removed (it doesn’t always).